src/header-bidding/slot-config.ts

```diff
@@ -198,6 +199,11 @@ const getSlots = (): HeaderBiddingSizeMapping => {
 			mobile: [adSizes.mpu],
 			desktop: [adSizes.billboard],
 		},
+		'article-end': {
+			mobile: isInUk() ? [adSizes.mpu] : [],
+			tablet: isInUk() ? [adSizes.mpu] : [],
+			desktop: isInUk() ? [adSizes.mpu] : [],
```

Comment:
Are these two lines necessary?

Reply:
We had just mobile at the beginning but I believe in Prebid we need to have each breakpoint and when we added them it worked when we tested it.